### ISSUE TITLE
When the mod is turned on, all particles disappear

### ISSUE THREAD

**Issue by DWolf-19** [SchmarrnDevs/Lighty#21]
# Video (unable to upload here (>10 MB))

https://drive.google.com/file/d/1uG-ZTNs00Foqa8jpaHu5qdubaJB9Y6Ve/view?usp=share_link

# Context 

- Windows 10
- Lighty 1.3.0

# Mods

![](https://github.com/SchmarrnDevs/Lighty/assets/108197792/4d42b079-4ddd-4b06-a75a-7c013b19fc53)

# Log (session when recording a video)

```
[00:09:57] [main/INFO]: Loading Minecraft 1.19.4 with Fabric Loader 0.14.19
[00:09:57] [main/INFO]: Loading 78 mods:
	- betterf3 6.0.1
	- cloth-config 10.0.96
	   \-- cloth-basic-math 0.6.1
	- cmdkeybind 1.6.2-1.19.4
	   \-- kyrptconfig 1.5.2-1.19.4
	- dynamicfps 2.2.0
	   \-- com_moandjiezana_toml_toml4j 0.7.2
	- entityculling 1.6.2-mc1.19.4
	- fabric-api 0.80.0+1.19.4
	   |-- fabric-api-base 0.4.25+504944c8f4
	   |-- fabric-api-lookup-api-v1 1.6.26+504944c8f4
	   |-- fabric-biome-api-v1 13.0.8+348a9c64f4
	   |-- fabric-block-api-v1 1.0.7+e022e5d1f4
	   |-- fabric-blockrenderlayer-v1 1.1.35+c2e6f674f4
	   |-- fabric-client-tags-api-v1 1.0.16+504944c8f4
	   |-- fabric-command-api-v1 1.2.28+f71b366ff4
	   |-- fabric-command-api-v2 2.2.7+504944c8f4
	   |-- fabric-commands-v0 0.2.45+df3654b3f4
	   |-- fabric-containers-v0 0.1.55+df3654b3f4
	   |-- fabric-content-registries-v0 3.5.10+ae0966baf4
	   |-- fabric-convention-tags-v1 1.4.2+9a7c5daaf4
	   |-- fabric-crash-report-info-v1 0.2.16+aeb40ebef4
	   |-- fabric-data-generation-api-v1 11.4.1+504944c8f4
	   |-- fabric-dimensions-v1 2.1.46+7f87f8faf4
	   |-- fabric-entity-events-v1 1.5.14+504944c8f4
	   |-- fabric-events-interaction-v0 0.5.0+76ba65ebf4
	   |-- fabric-events-lifecycle-v0 0.2.53+df3654b3f4
	   |-- fabric-game-rule-api-v1 1.0.34+a1ccd7bff4
	   |-- fabric-item-api-v1 2.1.18+504944c8f4
	   |-- fabric-item-group-api-v1 3.0.6+043f9acff4
	   |-- fabric-key-binding-api-v1 1.0.34+504944c8f4
	   |-- fabric-keybindings-v0 0.2.32+df3654b3f4
	   |-- fabric-lifecycle-events-v1 2.2.16+504944c8f4
	   |-- fabric-loot-api-v2 1.1.28+75e98211f4
	   |-- fabric-loot-tables-v1 1.1.32+9e7660c6f4
	   |-- fabric-message-api-v1 5.1.2+504944c8f4
	   |-- fabric-mining-level-api-v1 2.1.40+49abcf7ef4
	   |-- fabric-models-v0 0.3.31+504944c8f4
	   |-- fabric-networking-api-v1 1.3.2+504944c8f4
	   |-- fabric-networking-v0 0.3.42+df3654b3f4
	   |-- fabric-object-builder-api-v1 7.0.4+504944c8f4
	   |-- fabric-particles-v1 1.0.24+f1e4495bf4
	   |-- fabric-recipe-api-v1 1.0.9+a1ccd7bff4
	   |-- fabric-registry-sync-v0 2.2.1+504944c8f4
	   |-- fabric-renderer-api-v1 2.2.6+504944c8f4
	   |-- fabric-renderer-indigo 1.1.2+81e8c576f4
	   |-- fabric-renderer-registries-v1 3.2.39+df3654b3f4
	   |-- fabric-rendering-data-attachment-v1 0.3.29+afca2f3ef4
	   |-- fabric-rendering-fluids-v1 3.0.22+504944c8f4
	   |-- fabric-rendering-v0 1.1.42+df3654b3f4
	   |-- fabric-rendering-v1 2.1.2+504944c8f4
	   |-- fabric-resource-conditions-api-v1 2.3.2+e6c7d4eef4
	   |-- fabric-resource-loader-v0 0.11.3+1e1fb126f4
	   |-- fabric-screen-api-v1 1.0.46+504944c8f4
	   |-- fabric-screen-handler-api-v1 1.3.21+504944c8f4
	   |-- fabric-sound-api-v1 1.0.10+504944c8f4
	   |-- fabric-transfer-api-v1 3.1.2+504944c8f4
	   \-- fabric-transitive-access-wideners-v1 3.0.4+504944c8f4
	- fabricloader 0.14.19
	- ferritecore 5.2.0
	- iris 1.6.2
	   |-- io_github_douira_glsl-transformer 2.0.0-pre13
	   |-- org_anarres_jcpp 1.4.14
	   \-- org_antlr_antlr4-runtime 4.11.1
	- java 17
	- lambdynlights 2.3.0+1.19.4
	   |-- pride 1.2.0+1.19.4
	   \-- spruceui 4.2.0+1.19.4
	- lighty 1.3.0+1.19.4
	- lithium 0.11.1
	- minecraft 1.19.4
	- modmenu 6.2.2
	- reeses-sodium-options 1.5.0+mc1.19.4-build.72
	- replaymod 1.19.4-2.6.12
	- sodium 0.4.11+build.26
	- sodium-extra 0.4.19+mc1.19.3-build.101
	   |-- caffeineconfig 1.1.0+1.17
	   \-- crowdin-translate 1.4+1.19.3
[00:09:58] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.5 Source=file:/C:/Users/User/AppData/Roaming/.minecraft/libraries/net/fabricmc/sponge-mixin/0.12.4+mixin.0.8.5/sponge-mixin-0.12.4+mixin.0.8.5.jar Service=Knot/Fabric Env=CLIENT
[00:09:59] [main/INFO]: Compatibility level set to JAVA_17
[00:10:00] [main/INFO]: Loaded configuration file for Lithium: 114 options available, 1 override(s) found
[00:10:00] [main/WARN]: Reference map 'mixins.nonmmlauncher.replaymod.refmap.json' for mixins.nonmmlauncher.replaymod.json could not be read. If this is a development environment you can ignore this message
[00:10:00] [main/INFO]: Loaded configuration file for Sodium: 35 options available, 2 override(s) found
[00:10:01] [main/INFO]: Loaded configuration file for Sodium Extra: 28 options available, 0 override(s) found
[00:10:03] [main/WARN]: Force-disabling mixin 'alloc.blockstate.StateMixin' as rule 'mixin.alloc.blockstate' (added by mods [ferritecore]) disables it and children
[00:10:03] [main/WARN]: Force-disabling mixin 'features.gui.font.MixinGlyphRenderer' as rule 'mixin.features.gui.font' (added by mods [iris]) disables it and children
[00:10:03] [main/WARN]: Force-disabling mixin 'features.sky.MixinWorldRenderer' as rule 'mixin.features.sky' (added by mods [iris]) disables it and children
[00:10:04] [main/WARN]: Error loading class: net/optifine/render/ChunkVisibility (java.lang.ClassNotFoundException: net/optifine/render/ChunkVisibility)
[00:10:04] [main/WARN]: Error loading class: shadersmod/client/ShadersRender (java.lang.ClassNotFoundException: shadersmod/client/ShadersRender)
[00:10:04] [main/WARN]: Error loading class: net/optifine/shaders/ShadersRender (java.lang.ClassNotFoundException: net/optifine/shaders/ShadersRender)
[00:10:27] [Datafixer Bootstrap/INFO]: 180 Datafixer optimizations took 1011 milliseconds
[00:10:34] [Render thread/INFO]: Launching a new process to probe the system configuration!
[00:10:36] [Render thread/INFO]: OpenGL Vendor: Intel
[00:10:36] [Render thread/INFO]: OpenGL Renderer: Intel(R) HD Graphics 5500
[00:10:36] [Render thread/INFO]: OpenGL Version: 3.3.0 - Build 20.19.15.5063
[00:10:37] [Render thread/INFO]: Environment: authHost='https://authserver.mojang.com', accountsHost='https://api.mojang.com', sessionHost='https://sessionserver.mojang.com', servicesHost='https://api.minecraftservices.com', name='PROD'
[00:10:38] [Render thread/INFO]: Setting user: DWolf_19
[00:10:40] [Render thread/INFO]: [BetterF3] Loading...
[00:10:40] [Render thread/INFO]: [BetterF3] All done!
[00:10:41] [Render thread/INFO]: [Indigo] Different rendering plugin detected; not applying Indigo.
[00:10:41] [Render thread/INFO]: [LambDynLights] Initializing LambDynamicLights...
[00:10:41] [Render thread/INFO]: [LambDynLights] Configuration loaded.
[00:10:41] [Render thread/INFO]: Let there be Lighty
[00:10:41] [Worker-Main-1/INFO]: Checking mod updates...
[00:10:44] [ForkJoinPool.commonPool-worker-1/WARN]: [Iris Update Check] This version doesn't have an update index, skipping.
[00:10:44] [Render thread/INFO]: Backend library: LWJGL version 3.3.1 SNAPSHOT
[00:10:45] [Render thread/INFO]: Debug functionality is disabled.
[00:10:45] [Render thread/INFO]: DSA support not detected.
[00:10:45] [Render thread/INFO]: Shaders are disabled because enableShaders is set to false in iris.properties
[00:10:45] [Render thread/INFO]: Shaders are disabled
[00:10:46] [Render thread/WARN]: Static binding violation: PRIVATE @Overwrite method method_23182 in sodium.mixins.json:features.item.MixinItemRenderer from mod sodium cannot reduce visibiliy of PUBLIC target method, visibility will be upgraded.
[00:10:47] [Render thread/INFO]: Hardware information:
[00:10:47] [Render thread/INFO]: CPU: 4x Intel(R) Core(TM) i3-5005U CPU @ 2.00GHz
[00:10:47] [Render thread/INFO]: GPU: Intel(R) HD Graphics 5500 (Supports OpenGL 3.2.0 - Build 20.19.15.5063)
[00:10:47] [Render thread/INFO]: OS: Windows 10 (10.0)
[00:10:49] [Render thread/INFO]: Reloading ResourceManager: vanilla, Fabric Mods (BetterF3, CaffeineConfig, Cloth Config v10, Command Macros, Dynamic FPS, EntityCulling-Fabric, Fabric API, Fabric API Base, Fabric API Lookup API (v1), Fabric Biome API (v1), Fabric Block API (v1), Fabric BlockRenderLayer Registration (v1), Fabric Client Tags, Fabric Command API (v1), Fabric Command API (v2), Fabric Commands (v0), Fabric Containers (v0), Fabric Content Registries (v0), Fabric Convention Tags, Fabric Crash Report Info (v1), Fabric Data Generation API (v1), Fabric Dimensions API (v1), Fabric Entity Events (v1), Fabric Events Interaction (v0), Fabric Events Lifecycle (v0), Fabric Game Rule API (v1), Fabric Item API (v1), Fabric Item Group API (v1), Fabric Key Binding API (v1), Fabric Key Bindings (v0), Fabric Lifecycle Events (v1), Fabric Loot API (v2), Fabric Loot Tables (v1), Fabric Message API (v1), Fabric Mining Level API (v1), Fabric Models (v0), Fabric Networking API (v1), Fabric Networking (v0), Fabric Object Builder API (v1), Fabric Particles (v1), Fabric Recipe API (v1), Fabric Registry Sync (v0), Fabric Renderer API (v1), Fabric Renderer - Indigo, Fabric Renderer Registries (v1), Fabric Rendering Data Attachment (v1), Fabric Rendering Fluids (v1), Fabric Rendering (v0), Fabric Rendering (v1), Fabric Resource Conditions API (v1), Fabric Resource Loader (v0), Fabric Screen API (v1), Fabric Screen Handler API (v1), Fabric Sound API (v1), Fabric Transfer API (v1), Fabric Transitive Access Wideners (v1), Fabric Loader, Iris, Kyrpt Config, LambDynamicLights, Lighty, Lithium, Mod Menu, Pride Lib, Reese's Sodium Options, Replay Mod, Sodium, Sodium Extra, SpruceUI), file/Default-Dark-Mode-1.19.4-v1.4.0.zip, replaymod_lang
[00:10:50] [Render thread/INFO]: Creating pipeline for dimension OVERWORLD
[00:10:59] [Render thread/WARN]: Missing sound for event: minecraft:item.brush.brushing
[00:10:59] [Render thread/WARN]: Missing sound for event: minecraft:item.brush.brush_sand_completed
[00:10:59] [Render thread/WARN]: Missing sound for event: minecraft:item.goat_horn.play
[00:10:59] [Render thread/WARN]: Missing sound for event: minecraft:entity.goat.screaming.horn_break
[00:10:59] [Render thread/WARN]: Missing sound for event: minecraft:music.overworld.cherry_grove
[00:10:59] [Render thread/INFO]: OpenAL initialized on device OpenAL Soft on Ãðîìêîãîâîðèòåëü è íàóøíèêè (Realtek High Definition Audio)
[00:10:59] [Render thread/INFO]: Sound engine started
[00:10:59] [Render thread/INFO]: Created: 1024x512x4 minecraft:textures/atlas/blocks.png-atlas
[00:11:00] [Render thread/INFO]: Created: 256x256x4 minecraft:textures/atlas/signs.png-atlas
[00:11:00] [Render thread/INFO]: Created: 512x512x4 minecraft:textures/atlas/shield_patterns.png-atlas
[00:11:00] [Render thread/INFO]: Created: 512x512x4 minecraft:textures/atlas/banner_patterns.png-atlas
[00:11:00] [Render thread/INFO]: Created: 1024x1024x4 minecraft:textures/atlas/armor_trims.png-atlas
[00:11:00] [Render thread/INFO]: Created: 64x64x4 minecraft:textures/atlas/decorated_pot.png-atlas
[00:11:00] [Render thread/INFO]: Created: 256x256x4 minecraft:textures/atlas/chest.png-atlas
[00:11:00] [Render thread/INFO]: Created: 512x256x4 minecraft:textures/atlas/shulker_boxes.png-atlas
[00:11:00] [Render thread/INFO]: Created: 512x256x4 minecraft:textures/atlas/beds.png-atlas
[00:11:01] [Render thread/WARN]: Shader rendertype_entity_translucent_emissive could not find sampler named Sampler2 in the specified shader program.
[00:11:02] [Render thread/INFO]: Created: 256x256x0 minecraft:textures/atlas/particles.png-atlas
[00:11:02] [Render thread/INFO]: Created: 256x256x0 minecraft:textures/atlas/paintings.png-atlas
[00:11:02] [Render thread/INFO]: Created: 128x128x0 minecraft:textures/atlas/mob_effects.png-atlas
[00:12:26] [Render thread/INFO]: Loaded 7 recipes
[00:12:28] [Render thread/INFO]: Loaded 1179 advancements
[00:12:31] [Render thread/INFO]: Applied 0 biome modifications to 0 of 63 new biomes in 27.55 ms
[00:12:31] [Server thread/INFO]: Starting integrated minecraft server version 1.19.4
[00:12:31] [Server thread/INFO]: Generating keypair
[00:12:41] [Server thread/INFO]: Preparing start region for dimension minecraft:overworld
[00:12:54] [Render thread/INFO]: Preparing spawn area: 0%
[00:12:54] [Render thread/INFO]: Preparing spawn area: 0%
[00:12:54] [Render thread/INFO]: Preparing spawn area: 0%
[00:12:54] [Render thread/INFO]: Preparing spawn area: 0%
[00:12:54] [Render thread/INFO]: Preparing spawn area: 0%
[00:12:54] [Render thread/INFO]: Preparing spawn area: 0%
[00:12:54] [Render thread/INFO]: Preparing spawn area: 0%
[00:12:54] [Render thread/INFO]: Preparing spawn area: 0%
[00:12:54] [Render thread/INFO]: Preparing spawn area: 0%
[00:12:54] [Render thread/INFO]: Preparing spawn area: 0%
[00:12:54] [Render thread/INFO]: Preparing spawn area: 0%
[00:12:54] [Render thread/INFO]: Preparing spawn area: 0%
[00:12:54] [Render thread/INFO]: Preparing spawn area: 0%
[00:12:54] [Render thread/INFO]: Preparing spawn area: 0%
[00:12:54] [Render thread/INFO]: Preparing spawn area: 0%
[00:12:54] [Render thread/INFO]: Preparing spawn area: 0%
[00:12:54] [Render thread/INFO]: Preparing spawn area: 0%
[00:12:54] [Render thread/INFO]: Preparing spawn area: 0%
[00:12:54] [Render thread/INFO]: Preparing spawn area: 0%
[00:12:54] [Render thread/INFO]: Preparing spawn area: 0%
[00:12:54] [Render thread/INFO]: Preparing spawn area: 0%
[00:12:54] [Render thread/INFO]: Preparing spawn area: 0%
[00:12:54] [Render thread/INFO]: Preparing spawn area: 0%
[00:12:54] [Render thread/INFO]: Preparing spawn area: 0%
[00:12:54] [Render thread/INFO]: Preparing spawn area: 0%
[00:12:54] [Render thread/INFO]: Preparing spawn area: 0%
[00:12:54] [Render thread/INFO]: Preparing spawn area: 0%
[00:12:54] [Render thread/INFO]: Preparing spawn area: 0%
[00:12:55] [Render thread/INFO]: Preparing spawn area: 0%
[00:12:55] [Render thread/INFO]: Preparing spawn area: 0%
[00:12:56] [Render thread/INFO]: Preparing spawn area: 16%
[00:12:57] [Render thread/INFO]: Preparing spawn area: 37%
[00:12:57] [Render thread/INFO]: Preparing spawn area: 45%
[00:12:58] [Render thread/INFO]: Time elapsed: 16708 ms
[00:12:58] [Server thread/INFO]: Changing view distance to 20, from 10
[00:12:58] [Server thread/INFO]: Changing simulation distance to 10, from 0
[00:13:02] [Netty Local Client IO #0/INFO]: Singleplayer Recording is disabled
[00:13:02] [Server thread/INFO]: DWolf_19[local:E:6c8b14cb] logged in with entity id 35 at (-1448.4292999149927, -59.0, 2364.300000011921)
[00:13:02] [Server thread/INFO]: DWolf_19 joined the game
[00:13:03] [Render thread/INFO]: Started 1 worker threads
[00:13:04] [Render thread/INFO]: Loaded 305 advancements
[00:13:12] [Server thread/WARN]: Can't keep up! Is the server overloaded? Running 10790ms or 215 ticks behind
[00:13:31] [Server thread/WARN]: Can't keep up! Is the server overloaded? Running 3686ms or 73 ticks behind
[00:13:41] [Server thread/INFO]: Saving and pausing game...
[00:13:41] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Vanilla 1.19.4]'/minecraft:overworld
[00:13:42] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Vanilla 1.19.4]'/minecraft:the_end
[00:13:42] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Vanilla 1.19.4]'/minecraft:the_nether
[00:13:49] [Render thread/INFO]: Stopping worker threads
[00:13:49] [Render thread/INFO]: Started 1 worker threads
[00:13:49] [Render thread/INFO]: Flushed changes to Minecraft configuration
[00:13:52] [Server thread/INFO]: Changing view distance to 5, from 20
[00:14:11] [Server thread/INFO]: Saving and pausing game...
[00:14:12] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Vanilla 1.19.4]'/minecraft:overworld
[00:14:12] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Vanilla 1.19.4]'/minecraft:the_end
[00:14:12] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Vanilla 1.19.4]'/minecraft:the_nether
[00:14:30] [Server thread/INFO]: Saving and pausing game...
[00:14:30] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Vanilla 1.19.4]'/minecraft:overworld
[00:14:30] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Vanilla 1.19.4]'/minecraft:the_end
[00:14:30] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Vanilla 1.19.4]'/minecraft:the_nether
[00:15:28] [Server thread/INFO]: DWolf_19 lost connection: Disconnected
[00:15:28] [Server thread/INFO]: DWolf_19 left the game
[00:15:28] [Server thread/INFO]: Stopping singleplayer server as player logged out
[00:15:28] [Server thread/INFO]: Stopping server
[00:15:28] [Server thread/INFO]: Saving players
[00:15:28] [Server thread/INFO]: Saving worlds
[00:15:32] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Vanilla 1.19.4]'/minecraft:overworld
[00:15:32] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Vanilla 1.19.4]'/minecraft:the_end
[00:15:32] [Server thread/INFO]: Saving chunks for level 'ServerLevel[Vanilla 1.19.4]'/minecraft:the_nether
[00:15:32] [Server thread/INFO]: ThreadedAnvilChunkStorage (Vanilla 1.19.4): All chunks are saved
[00:15:32] [Server thread/INFO]: ThreadedAnvilChunkStorage (DIM1): All chunks are saved
[00:15:32] [Server thread/INFO]: ThreadedAnvilChunkStorage (DIM-1): All chunks are saved
[00:15:32] [Server thread/INFO]: ThreadedAnvilChunkStorage: All dimensions are saved
[00:15:32] [Render thread/INFO]: Stopping worker threads
[00:15:39] [Render thread/INFO]: Stopping!
[00:15:39] [CullThread/INFO]: [STDOUT]: Shutting down culling task!
```

**Comment by andi-makes** [comment]
Thank you for opening this issue, I was able to reproduce the bug.

It may take some time until I've fixed it though because life is busy right now.

**Comment by andi-makes** [comment]
Nevermind "it will take some time", I was able to find the bug rather quickly and released [Lighty 1.3.1 containing the fix to Modrinth](https://modrinth.com/mod/lighty/version/v1.3.1+1.19.4).

Again, thank you for reporting the issue and have fun with the new Lighty version!
